course7/homeworks/kushagra/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(my_ls)
+
+find_package(Boost 1.55 REQUIRED COMPONENTS filesystem system)
+
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fexceptions -Wextra -std=c++14")
+if(Boost_FOUND)
```

Comment:
Why the if if you mark Boost as REQUIRED in find_package above. If Boost is not found, configuration will fail altogether.